### ISSUE TITLE
Too many icons in the Search title area

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#188781]
My feeling is that there are too many icons (that are similar) in the Search view title area.

Could we check in telemetry how much each of these buttons is popular?
And once we do that, does it make sense to introduce a `...` button at the end, and we could move the "View as Tree" there (once data probably confirms this is not so popular).
The `...` could be a place where we introduce other actions. For example some connection to our "/search" experience.

![Screenshot 2023-07-25 at 15 22 24](https://github.com/microsoft/vscode/assets/1926584/65f665f1-78be-4513-8f37-e5275a5b660c)

**Comment by daviddossett** [comment]
I could imagine the view mode and the new search editor buttons being candidates for a `...` menu. But I don't have the data to back that up 🤔

**Comment by isidorn** [comment]
Here are the numbers of those actions triggered in past 28 days from exactly that toolbar:
* Refresh: 349K
* Clear: 738K
* Open New Search Editor: 209K
* viewAsTree: 47.9K viewAsList 23.9K (it is a toggle). Total = 71.8K
* Collapse: 2.2M

Thus I suggest we move the viewAsTree and viewAsList into the `...`. Those numbers are small compared to the other actions.

What do you think?

**Comment by andreamah** [comment]
Yeah, I think that makes sense. We can consider this again when we have more things to put in the `...` list 👍

**Comment by andreamah** [comment]
There's also a conversation to be had about where the `...` would be. Ideally, we could put it on the far right side, but lots of people expect the far right icon to be collapse/expand icon.

**Comment by daviddossett** [comment]
I agree, it would be strange to have the menu anywhere else, at least in a typical L->R layout. Worth the muscle memory tradeoff IMO.

**Comment by andreamah** [comment]
Yeah, I agree with that. There's probably more muscle memory regarding having a "..." menu on the far right anyways.
